jgiven-gradle-plugin/src/main/java/com/tngtech/jgiven/gradle/internal/AbstractJGivenReportImpl.java

```diff
@@ -64,7 +64,7 @@ public AbstractReportGenerator createGenerator() {
                     customConf.setCustomJs( getCustomJsFile() );
                 }
                 conf = customConf;
-                generator = ReportGenerator.generateHtml5Report();
+                generator = ReportGenerator.loadHtml5ReportGenerator();
```

Comment:
why don't we need to adapt the asciidoc report here?

Reply:
I mean... we can just use the constructor here...